### DISCUSSION TITLE
Can't connect Renovate to the repo.

### DISCUSSION THREAD

**Discussion by dangreen** [renovatebot/renovate#19124] [Request Help] [answered]
Hi. I am the owner of the repo [browserslist-useragent-regexp](https://github.com/browserslist/browserslist-useragent-regexp). When I asked the org admin to add Renovate, he received this error: "Sorry, there may be an issue with your account. We're looking into it and will get back to you within 48 hours. Thanks for your patience." 48 hours are already left.

**Comment by rarkins** [comment, marked as answer]
Where did he receive the error from? That's not an error I've heard of before, perhaps it was by GitHub? Please ask for a screenshot so that it's more clear.

**Reply by dangreen** [reply]
@rarkins 

https://user-images.githubusercontent.com/986756/204384614-ebc59eae-1df6-43cd-895a-ccbc7a0d0f8d.mp4

**Reply by rarkins** [reply]
That's a GitHub message. I think it may happen for accounts which are in invoice billing.

Good news is you can skip that page and instead just install directly from https://github.com/apps/renovate

**Reply by dangreen** [reply]
@rarkins Thanks! It works.
